### ISSUE TITLE
Installation issue - Missing closing bracket in feature-helpers.R

### ISSUE THREAD

**Issue by haas-christian** [aaronrkaufman/compactness#10]
Hello,

First of all, thank you for this nice package. When I tried to install it, I received the error message 
```
Error in parse(...) : 
  C:/../compactness/R/feature_helpers.R:83:3: unexpected symbol
82:   temp = lapply(1:nrow(shp[[1]]), FUN=function(x) get_one_bound_feature(x)
83:   out
      ^
ERROR: unable to collate and parse R files for package 'compactness'
```

Upon closer inspection, this is due to a missing closing bracket in line 82 of the feature_helpers.R file. When I downloaded the compactness package, manually changed the feature_helpers.R file, and then installed from the local version, it worked.

So, I'd suggest the following change for line 82 of feature_helpers.R (note the double closed bracket for the first line in the function):

```
get_all_bound_features = function(shp){  
      temp = lapply(1:nrow(shp[[1]]), FUN=function(x) get_one_bound_feature(x))  
      out = do.call(rbind, temp)  
      return(out) 
}
```
Just thought this might be helpful if other users have the same issue. Thanks!

**Comment by aaronrkaufman** [comment]
Thanks! Fixed. The package is currently under construction a bit (once I get out from under start of semester madness) to add some new features!

**Comment by haas-christian** [comment]
Thanks! Looking forward to the additional features.

Unfortunately, I just realized that even though the bracket is now closed and the installation works, line 82 causes a different issue now: 
```
Error in shp[[2]][x] : invalid subscript type 'list'
```
This seems to come from the get_all_bound_features function, calling get_one_bound_feature on line 82.

**Comment by aaronrkaufman** [comment]
Yes, I see! Take a look at the newest version. Note that you may need to set the following global option: 

units::units_options(allow_mixed=T)
